org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -174,6 +177,27 @@ public void start(BundleContext bundleContext) throws Exception {
 		if (System.getProperty(AssistOptions.PROPERTY_SubstringMatch) == null) {
 			System.setProperty(AssistOptions.PROPERTY_SubstringMatch, "false");
 		}
+		// https://github.com/redhat-developer/vscode-java/issues/3797
+		Job initializeAfterLoad = new Job("Initialize After Load") {
+
+			@Override
+			protected IStatus run(IProgressMonitor monitor) {
+				setPriority(Job.SHORT);
```

Comment:
Does this have any effect ? `setPriority` should affect scheduling but at this point the job has already started running.

Reply:
Fixed.